testutil/validatormock/synccomm.go

```diff
@@ -47,7 +47,7 @@ func NewSyncCommMember(eth2Cl eth2wrap.Client, epoch eth2p0.Epoch, signFunc Sign
 		dutiesOK:     make(chan struct{}),
 		selections:   make(map[eth2p0.Slot]syncSelections),
 		selectionsOK: make(map[eth2p0.Slot]chan struct{}),
-		blockRoot:    make(map[eth2p0.Slot]*eth2p0.Root),
```

Comment:
changing this it non-pointer since basic eth2 types aren't pointers and this causes a panic when duties time out.